galaxy/main/models/repository.py

```diff
@@ -106,7 +106,18 @@ class Meta:
 
     @property
     def clone_url(self):
-        return "https://github.com/{user}/{repo}.git".format(
+        server = ''
+        if 'github' in settings.SOCIALACCOUNT_PROVIDERS:
+            if 'GITHUB_URL' in settings.SOCIALACCOUNT_PROVIDERS['github']:
+                server = settings \
+                    .SOCIALACCOUNT_PROVIDERS['github'] \
+                    .GITHUB_URL
+            else:
```

Comment:
Same here. Also this code seems to be duplicate of code above.

Reply:
Fixed in last commit